tensorboard/plugins/mesh/http_api.md

```diff
@@ -49,23 +49,26 @@ Here is an example:
             "data_shape": [1, 17192, 3],
             "step": 0,
             "content_type": 2,
-            "wall_time": 1556678491.836787
+            "wall_time": 1556678491.836787,
+            "components": 14
```

Comment:
Please document this—a short sentence would suffice, but right now it
looks like this means that the summary has 14 components, which is not
correct.